frontend/src/router.tsx

```diff
@@ -25,6 +25,10 @@ import EditAgencyPage from 'features/admin/agencies/EditAgencyPage';
 import { IENotSupportedPage } from 'features/account/IENotSupportedPage';
 
 const AppRouter: React.FC = () => {
+  const location = useLocation();
```

Comment:
Snuck this in here: this ensures that changing the route loads the top of the newly loaded page.